tests/tensorflow_autolog/test_tensorflow_autolog.py

```diff
@@ -257,16 +257,13 @@ def test_tf_keras_autolog_early_stop_logs(tf_keras_random_data_run_with_callback
     assert "restored_epoch" in metrics
     assert "epoch_loss" in metrics
     restored_epoch = int(metrics["restored_epoch"])
-    assert int(metrics["stopped_epoch"]) - max(1, callback.patience) == restored_epoch
+    assert int(metrics["stopped_epoch"]) - callback.patience == restored_epoch
     assert "loss" in history.history
-    num_of_epochs = len(history.history["loss"])
     client = mlflow.tracking.MlflowClient()
     # TF 1.X TB callback logs loss as `epoch_loss`
     metric_history = client.get_metric_history(run.info.run_id, "epoch_loss")
-    # Check the test epoch numbers are correct
-    assert num_of_epochs == max(1, callback.patience) + 1
```

Comment:
The number of epochs resulting from early stopping varies across tensorflow versions; we do not need to assert on it since it is a property of TensorFlow and not of MLflow.